proposals/wac-ucr/index.bs

```diff
@@ -1114,13 +1114,178 @@ and the presentation goes off perfectly.
 Requirements {#requirements}
 ================================================================================
 
-Issue: Populate requirements based on established use cases.
+## Functional Requirements ## {#req-functional}
+
+1. The system shall permit or restrict an [=agent's=] ability to access a 
+    [=resource=] or [=collection=]. [[#uc-basic]], 
+    [[#uc-collections]], [[#uc-blockagents]]
+
+1. The system shall allow access to be limited based on the [=identity=] of 
+    the [=agent=]. [[#uc-basic]], [[#uc-collections]]
+
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the [=agent's=] membership in a certain group of [=agents=]. 
+    [[#basic-group]]
+    
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the client [=application=] in use by the [=agent=]. 
+    [[#uc-trusted-applications]]
+
+1. The system shall allow for access to a certain resource to be limited to 
+    the [=agent=] that created that resource.
+    [[#collection-readappendwrite]]
+
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the [=agent's=] possession of a certain verifiable credential
+    or capability. FUTURE. 
+    [[#capabilities-vc]], [[#capabilities-link]], [[#uc-minimalcredentials]]
+    
+1. The system shall allow access to be limited to any 
+    unauthenticated [=agent=].
+    [[#basic-public]]
+    
+1. The system shall allow access to be limited to any 
+    authenticated [=agent=].
+    [[#basic-authenticated]]
+
+1. The system shall limit the ability to read the access permissions 
+    associated with a certain [=resource=].
+    [[#basic-control]], [[#uc-whopermitted]], [[#uc-historyofchanges]]
+    
+1. The system shall limit the ability to change the access permissions 
+    associated with a certain [=resource=].
+    [[#basic-control]]
+
+1. The system shall furnish the effective access permission modes on a certain
+    [=resource=] as they relate to the [=agent=] making the request,
+    in the request response.
+    [[#uc-client-determine-access-privileges]]
+
+1. The system shall limit the ability to read a certain [=resource=].
+    [[#basic-readonly]], [[#basic-write]], [[#basic-readappend]]
+
+1. The system shall limit the ability to change any of the contents of a 
+    certain [=resource=].
+    [[#basic-write]]
```

Comment:
Saying "_any of the contents of a certain resource_" would exclude append.

Reply:
I don't follow. Can you break it down.. express with more common terms (instead of "change") on what's intended. Create/Update/Delete? Replace? Or even the generic "Write". I don't see how "change" excludes "append".

Reply:
Expand upon _any_ (e.g. create, update, delete)

Reply:
maybe just adding *existing* would clarify append?

> ability to change any of the *existing* contents

Reply:
maybe just adding *existing* would clarify append?

> ability to change any of the *existing* contents

Reply:
Clarified in https://github.com/solid/authorization-panel/pull/152/commits/32b5bb6501b5234326d0e06d726f152bee6a88bd per @elf-pavlik's suggestion.